.github/workflows/tag.yaml

```diff
@@ -0,0 +1,36 @@
+name: Create tag
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - closed
+    paths-ignore:
+      - LICENSE
+      - README.md
+      - .github/**
+      - .*
+
+permissions:
+  contents: read
+  packages: read
+
+jobs:
+  tag:
+    name: Tagging
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
What scopes does the `PERSONAL_ACCESS_TOKEN` token need? I assume only `public_repo`?

Can we use `secrets.GITHUB_TOKEN` here instead?

Reply:
Sorry, that was a left over from where I copied it. We don't need any extra token for that. I have removed it in the last commit.